src/python/WMCore/WMBS/MySQL/Files/GetLocationBulk.py

```diff
@@ -11,11 +11,10 @@
 from WMCore.Database.DBFormatter import DBFormatter
 
 class GetLocationBulk(DBFormatter):
-    sql = """SELECT wls.pnn AS pnn, wfl.fileid AS id FROM wmbs_location wl
-                INNER JOIN wmbs_file_location wfl ON wfl.location = wl.id
-                INNER JOIN wmbs_location_pnns wls ON wls.location = wl.id
-                WHERE wfl.fileid = :id
-                """
+    sql = """SELECT wfl.location as pnn, wfl.fileid as id
+               FROM wmbs_file_location wfl
+               WHERE wfl.fileid = :id
+            """
```

Comment:
If you need to return the same thing it should be  pnn string not pnn integer.
need to join wmbs_pnns table.